hugegraph-dist/src/assembly/static/bin/hugegraph

```diff
@@ -37,6 +37,7 @@
 ### END INIT INFO
 
 # Variables
+# it requires user to set a fixed abs path manually
```

Comment:
we can try `${INSTALL_DIR:?"Please setting variables 'INSTALL_DIR'"}`
for more details: https://stackoverflow.com/a/16753536

Reply:
> https://stackoverflow.com/a/16753536
@javeme seems a little strange, could it use directly or test it in local evn? (or should in a if condition?)

<img width="200" alt="image" src="https://user-images.githubusercontent.com/17706099/204548182-09d65cc4-d721-4b7b-81ef-0920ddbd77db.png">

and it alert:
<img width="331" alt="image" src="https://user-images.githubusercontent.com/17706099/204548268-ffab90a5-7d4a-48ea-87bc-5e9300ffa3d2.png">

failed all cases in my ubuntu21